packages/react/src/global-nav/GlobalNav.tsx

```diff
@@ -0,0 +1,83 @@
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+import { useControllableState } from "@radix-ui/react-use-controllable-state";
+import { type ComponentPropsWithRef, forwardRef } from "react";
+import { type ReactNode } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box, type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { GlobalNavContext } from "../global-nav-context";
+import { GlobalNavItem } from "../global-nav-item";
+import { IconCollapse } from "../icons/IconCollapse";
+import { extractSprinkles } from "../sprinkles";
+
+type GlobalNavProps = ExtendProps<
+  Omit<
+    ComponentPropsWithRef<typeof RadixCollapsible.Root>,
+    "defaultOpen" | "onOpenChange" | "open"
+  >,
+  BoxProps<
+    "nav",
+    {
+      defaultExpanded?: boolean;
+      endDecorator?: ReactNode;
+      expanded?: boolean;
+      onExpandedChange?: (expanded: boolean) => void;
+    }
+  >
+>;
+
+export const GlobalNav = forwardRef<HTMLElement, GlobalNavProps>(
+  (
+    {
+      children,
+      defaultExpanded,
+      endDecorator,
+      expanded: expandedProp,
+      onExpandedChange,
+      ...props
+    },
+    ref,
+  ) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    const [expanded, setExpanded] = useControllableState({
+      defaultProp: defaultExpanded,
+      onChange: onExpandedChange,
+      prop: expandedProp,
+    });
+
+    return (
+      <GlobalNavContext.Provider value={{ expanded }}>
+        <RadixCollapsible.Root onOpenChange={setExpanded} open={expanded}>
+          <Flex border="1" pb="md" pt="lg">
+            <Box asChild display="flex" {...sprinkleProps}>
+              <nav
+                aria-label="Global Navigation"
+                ref={ref}
+                role="navigation"
```

Comment:
It will cause redundancy for screen readers. 

```
Screen readers will announce the type of role the landmark is. Because of this, you do not need to describe what the landmark is in its label. For example, a declaration of role="navigation" with an of aria-label="Primary navigation" may be announced redundantly as, "primary navigation navigation".
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role